docs/workforce-management-framework/topics/con-wfm-extending-user-interface.adoc

```diff
@@ -14,7 +14,7 @@ This Module contains the example directives that are used for demo purposes and
 By default, {Feature} provides a minimal set of defined fields for a _workorder_.
 Depending on the needs of the business, a developer can add additional data and UI elements, for example, gps location, an address or a description UI element.
 
-=== Building Workorders
+=== Building
```

Comment:
Once again - extending not building :)

Reply:
I have added - building directives. Should be ok.

Reply:
@wtrocki - can you take a look back though the history here - I think you are confused and you are, being honest,  causing me to become _very_ confused and I don't appreciate it. 

1) This heading was originally : "=== Building"
2) I changed it to : "=== Building Workorders"
3) You made the comment: "Extending Workorders != Budiling Workorders. Can we revert?"
4) I reverted it to original, as per your comment: "=== Building"
5) You then add the comment: "Once again - extending not building :)"
6) You then change it to: "=== Building directives"

Reply:
I think that git changes got us confused - myself especially. 
TLDR;

- Building was ok but were incomplete
- Building workorder were missleading as were building extension directives here.
- Extending workorder will be better but.. it's already mentioned on the top
- Building directives - was my final candidate as our target was to extend this to provide info what we going to build.
made change and merged that.